.github/workflows/asana-release.yml

```diff
@@ -1,25 +1,193 @@
-name: Asana Release Workflow
+name: Autofill Release Workflow
 
 on:
   release:
     types: [ created ]
 
 jobs:
-  asana_release:
+  autofill_release: # TODO: rename the file to autofill-release.yml once reviewed to keep a clear diff
     runs-on: ubuntu-20.04
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js 16
-      uses: actions/setup-node@v1
-      with:
-        node-version: 16.x
-    - name: Asana Workflow
-      env:
-        ASANA_ACCESS_TOKEN: ${{ secrets.NATIVE_APPS_WORKFLOW }}
-        VERSION: ${{ github.event.release.tag_name }}
-        RELEASE_URL: ${{ github.event.release.html_url }}
-        RELEASE_NOTES: ${{ github.event.release.body }}
-      run: |
-        npm install
-        node scripts/release.js
+      - uses: actions/checkout@v3
+        with:
+          path: autofill/
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Create Asana Tasks
+        id: create-asana-tasks
+        env:
+          ASANA_ACCESS_TOKEN: ${{ secrets.NATIVE_APPS_WORKFLOW }}
+          VERSION: ${{ github.event.release.tag_name }}
+          RELEASE_URL: ${{ github.event.release.html_url }}
+          RELEASE_NOTES: ${{ github.event.release.body }}
+        run: |
+          npm install
+          JSON_STRING="$(node ./autofill/scripts/release/asana-create-tasks.js)"
+          echo "ASANA_OUTPUT=$JSON_STRING" >> $GITHUB_OUTPUT
+      - name: Checkout Android
+        uses: actions/checkout@v3
+        with:
+          repository: duckduckgo/android
+          path: android/
+          ref: develop
+          token: ${{ secrets.DAXMOBILE_AUTOFILL_AUTOMATION }}
+      - name: Update Android autofill reference
+        env:
+          BSK_SHA: ${{ steps.bsk-pr.outputs.pull-request-head-sha }}
+        run: |
+          node ./autofill/scripts/release/update-android-repo.js
+          cd android
+          npm install
+          cd ..
+      - name: Create Android PR Body
+        env:
+          VERSION: ${{ github.event.release.tag_name }}
+          RELEASE_URL: ${{ github.event.release.html_url }}
+          RELEASE_NOTES: ${{ github.event.release.body }}
+          ASANA_OUTPUT: ${{ steps.create-asana-tasks.outputs.ASANA_OUTPUT }}
+        run: |
+          TEMPLATE="$(node ./autofill/scripts/release/create-pr-template.js android)"
+          echo "PR_BODY_ANDROID<<EOF" >> $GITHUB_ENV
+          echo "$TEMPLATE" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
```

Comment:
This is how you set multiline variables to env. I will open a security review for the delimiter because there's a warning on the [doc page](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings).